src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -103,6 +104,27 @@
 		return true;
 	}
 	const { t, locale } = useI18n();
+	onMount(() => {
```

Comment:
Why is this in `onMount`?

Google shopping needs the info to be avaialble at first render of the page